example/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "ReactNativeImageEditorExample",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test:android:test:debug": "detox test -c android.emu.debug",
+    "test:android:test:release": "detox test -c android.emu.release",
+    "test:android:build:debug": "detox build -c android.emu.debug",
+    "test:android:build:release": "detox build -c android.emu.release",
+    "test:android:bundle:release": "mkdir -p .tmp && react-native bundle --max-workers 4 --platform android --dev false --entry-file index.js --bundle-output .tmp/android-bundle.js",
+    "test:ios:test:debug": "detox test -c ios.sim.debug",
```

Comment:
We don't need this file. Example app should use same dependencies as library. Check my first comment for more information